unstructured/partition/pdf_image/ocr.py

```diff
@@ -8,9 +8,11 @@
 # unstructured.documents.elements.Image
 from PIL import Image as PILImage
 from PIL import ImageSequence
+from unstructured_inference.models.tables import cells_to_html
```

Comment:
in `unstructured` we use a decorator on functions to note if this function requires additional extras to run. And `unstructured_inference` is one of those extras. As a result we don't import from `unstructured_inference` at module level normally but import only where it is needed and decorate the function with `@requires_dependencies("unstructured_inference")`. You can find examples in this file.